doc/build.md

```diff
@@ -175,20 +190,22 @@ e.g
 python build_usd.py ~/Desktop/BUILD --build-args boost,"--with-date_time --with-thread --with-system --with-filesystem" 
 ```
 
+***NOTE:*** ```--build-args``` needs to be passed at the very end of command, after build/install location.
+
 ##### Python:
 
-Some DCCs (most notably, Maya) may ship with and run using their own version of Python. In that case, it is important that USD and the plugins for that DCC are built using the DCC's version of Python and not the system version. Note that this is primarily an issue on macOS, where a DCC's version of Python is likely to conflict with the version provided by the system. 
+It is important to use the Python version shipped with Maya and not the system version when building USD on MacOS. Note that this is primarily an issue on MacOS, where Maya's version of Python is likely to conflict with the version provided by the system. 
 
-To build USD and the Maya plug-ins on macOS for Maya(2018,2019,2020), run:
+To build USD and the Maya plug-ins on MacOS for Maya(2018,2019,2020), run:
```

Comment:
@mattyjams  I changed the wording a little bit and hopefully it makes more sense?

Reply:
Works for me, thanks!